lib/semver.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "@nicolo-ribaudo/semver-v6" {
+  export { default } from "./semver";
+  export * from "./semver";
```

Comment:
This is a "circular dependency", it should re-export from the real `semver` package. If TS gives an error about `semver` not being found, you can add semver@7 as a devDependency of the top-level package.json

Reply:
The change doesn't seem to bother `tscheck`